tools/configure_test_cluster.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python3
+
+import logging
+import os
+import os.path
+import subprocess
+import sys
+
+import venvutil
+
+# Things this needs: cluster_id, stack_id (???), auth_token, dns_address,
```

Comment:
The `cluster_id` is a sequential int made up by CCM, while the `stack_id` is the AWS cloudformation stack (colon-delimited string iirc).
The variable names below could be updated to `ccm_cluster_id` and `aws_stack_id` to clarify the difference a bit.